### PR TITLE
Updating to pytorch-transformers

### PR DESCRIPTION
**Pull request by desireevl**
Fix to #14. I believe that this should update the things that have changed from the migration from pretrainedbert to pytorch-transformers. This update runs fine for me using the toxic Kaggle dataset however, I think there may be some other things in trainer.py that I have missed.

### CODE REVIEW

**Comment by maxware0715** [comment]
Thank you!
By the way, focal loss is better than BCE loss, especially for hundreds of labels
